test/test_grocy.py

```diff
@@ -218,6 +218,20 @@ def test_consume_product_error(self):
         )
         self.assertRaises(GrocyError, self.grocy.consume_product, 1, 1.3)
 
+    @responses.activate
```

Comment:
please convert these to vcr casette based tests and move them to `test_stock.py`

Reply:
Done! Thanks!